cpp/util/Histogram.h

```diff
@@ -133,11 +157,7 @@ class RegularAxis : public Axis
 #else
         unsigned int bin = (unsigned int)(val);
 #endif
-        // Avoid rounding leading to overflow.
```

Comment:
So is this rounding issue no longer a problem?

Reply:
I've never actually hit it, yeah. The current check was actually wrong, so I just removed it. With ManagedArray it should be easy to diagnose if it actually ever comes up, though.